pudb/debugger.py

```diff
@@ -2613,7 +2614,8 @@ def make_frame_ui(frame_lineno):
                 try:
                     class_name = frame.f_locals["self"].__class__.__name__
                 except Exception:
-                    pass
+                    from pudb.lowlevel import ui_log
+                    ui_log.exception('Failed to determine class name')
```

Comment:
`class_name` should perhaps be updated to reflect that an error occurred when trying to find it.